src/monitor.ts

```diff
@@ -41,11 +41,13 @@ function validateMonitoredNodeConfig(data: any): IMonitoredNodeConfig {
     throw new Error('address is not a string');
   }
 
-  if (typeof(data.reviveCmd) !== 'string') {
+  if (data.reviveCmd &&
```

Comment:
so it's ok if data.revivecmd is null or undefined?

Reply:
Yeah, the idea is that we can revive remote nodes, but still want to monitor them